src/.htaccess

```diff
@@ -84,7 +84,6 @@ redirect permanent /x https://xhr.spec.whatwg.org/
 redirect permanent /xhr https://xhr.spec.whatwg.org/
 
 redirect permanent /newbug https://github.com/whatwg/html/issues/new
-redirect permanent /faq https://wiki.whatwg.org/wiki/FAQ
```

Comment:
I was hoping that /faq it was rarely linked, but https://html.spec.whatwg.org/ does. But, I'd still like to use the shorter URL, any breakage will be transient.